build/source/dshare/var_lookup.f90

```diff
@@ -24,8 +24,9 @@ MODULE var_lookup
  implicit none
  private
  ! local variables
- integer(i4b),parameter     :: imiss = -999             ! missing value: used to initialize named variables
- integer(i4b),parameter     :: iLength=storage_size(1)  ! size of an integer
+ integer(i4b),parameter     :: imiss = -999                 ! missing value: used to initialize named variables
```

Comment:
This is tricky. The routine `globalData.f90` actually uses the module `var_lookup`, and it would require some reorganization to define missing values that can be used in `var_lookup`.

Ultimately we may want two forms of global data: (1) global constants (e.g., put global variables in `multiconst.f90`); and (2) global data structures (e.g., the metadata structures). This would enable us to use `integerMissing` in `var_lookup`

This extent of reorganization goes beyond what is attempted here, and is left for a future refactor.